addons/l10n_hu_edi/wizard/l10n_hu_edi_tax_audit_export.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields, _
+from odoo.exceptions import UserError
+
+import base64
+import tempfile
+import zipfile
+import contextlib
+
+class L10nHuEdiTaxAuditExport(models.TransientModel):
+    _name = 'l10n_hu_edi.tax_audit_export'
+    _description = 'Tax Audit Export Wizard'
```

Comment:
Does not seem very "Tax Audit".  More like a way to download all the XMLs linked to the invoices?